_tests/browser_tests/spec_suites/large_screen/large_contact-us.js

```diff
@@ -0,0 +1,23 @@
+ContactUsPage = require( '../../page_objects/page_contact-us.js' );
+
+describe( 'Beta Contact Page', function() {
+  var page;
+
+  beforeEach( function() {
+    page = new ContactUsPage();
+    page.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Contact us' );
+  } );
+
+  it( 'should include 48 individual offices in alpha order', function() {
+    var firstOffice = page.offices.first().element( by.css( '.expandable_label' ) );
+    var lastOffice = page.offices.last().element( by.css( '.expandable_label' ) );
```

Comment:
Ideally all code that is doing some kind of element selection should be in Page Object. The specs should read like a book and contain any lower level concerns of implementation details In this case your spec "knows" the css class of the expandable.